<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Video2Pictures&amp;process</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="selectVideo">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>选择视频</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startCut">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>开始剥离</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="videoPath">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="pictureOutPath">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>图片输出路径</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectOutputPath">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>选择输出路径</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>源视频</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>421</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>90</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="startProcess">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>210</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>开始数据处理</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>53</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>帧间隔</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FPS">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>20</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cloudOutPath">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>点云输出路径</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectCloudOutPath">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>170</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>选择输出路径</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="speed">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>220</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>0.05</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>220</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>m/s（可正负）</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_pushDirction">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;激光雷达设备： | |========\ \ &lt;/p&gt;&lt;p&gt;X方向为 平行于激光雷达摆臂的方向&lt;/p&gt;&lt;p&gt;Y方向为 平行于旋转电机的方向&lt;/p&gt;&lt;p&gt;Z方向为 垂直于激光雷达摆臂的方向，与激光器仓平行&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <item>
     <property name="text">
      <string>X方向</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Y方向</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Z方向</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="PZAngle">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>260</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>260</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>度（可正负）</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>偏转角度(基于所选方向)</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
